act/io/armfiles.py

```diff
@@ -192,20 +265,15 @@ def create_obj_from_arm_dod(proc, set_dims, version='', fill_value=-9999.,
         Depending on how the object is set up, sometimes the scalar values
         are dimensioned to the main dimension.  i.e. a lat/lon is set to have
         a dimension of time.  This is a way to set it up similarly.
-
```

Comment:
I believe this spaces are needed for sphinx to recognize.

Reply:
@kenkehoe I just noticed where Zach's comment was.  In the docstrings, you removed the space between some of the sections.  Can you check and add those spaces back in as appropriate?